redhat-cop/casl-ansible
# Adding pre/post steps for openshift-applier

#### What does this PR do?
This PR introduces the pre/post steps for the openshift-applier. This has been a feature long coming, but due to a bug in ansible, it has not been possible to complete till now.

As the fix for Ansible is not available til release 2.5, we have created a docker image that can be used to run the `openshift-applier` with ansible 2.5. This image is being delivered by PR https://github.com/redhat-cop/casl-ansible/pull/154. 

#### How should this be manually tested?
Follow the documentation in the README, and add pre/post steps to the inventory. Then use the container to run it (see instructions in the container README).

#### Is there a relevant Issue open for this?
resolves #133 

#### Who would you like to review this?
cc: @redhat-cop/casl
